dso/model.py

```diff
@@ -118,8 +119,18 @@ class MalwareSummary:
     findings: list[clamav.client.ScanResult]
 
 
+@dataclasses.dataclass(frozen=True)
+class Codecheck:
```

Comment:
naming: rather call this CodecheckResult, or CodecheckSummary (similar to MalwareSummary), or CodecheckFindings

as for the attributes, rather name those, e.g. high_findings. or add an attribut `findings`, like so:
```
findings:
 high
 medium
 low
 info
```

Reply:
also, I just saw you already did change those names in the second commit from this pull-request.. please mv the diffs between commits accordingly